client/dive-common/use/useModeManager.ts

```diff
@@ -377,11 +379,11 @@ export default function useModeManager({
               selectedKey.value,
               annotationModes.editing,
             );
+            handleSelectFeatureHandle(Math.max(selectedFeatureHandle.value - 1, 0));
```

Comment:
I guess really the only way to do it would be to get the `selectedTrack` current geoJSON geometry for the editing type. get the length of the coordinates array and then use that.  It would definitely be a pain to get that information just for this.

Or perhaps you could make it the return value of deletePoint in the recipe?
The recipe already has this snippet:
```
track.getFeatureGeometry(frame, {
        type: 'Polygon',
        key: '',
      });
```
Allowing you to get the geoJSON data and then determine what point should be selected next after deletion?